test/e2e-tests.sh

```diff
@@ -30,10 +30,13 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 # Names of the Resources used in the tests.
 readonly E2E_TEST_NAMESPACE=e2etest
 readonly E2E_TEST_FUNCTION_NAMESPACE=e2etestfn3
+readonly ISTIO_CRD_YAML="${KNATIVE_BASE_YAML_SOURCE/@/serving}/istio-crds.yaml"
```

Comment:
This won't work as you expect, since KNATIVE_BASE_YAML_SOURCE is not defined.
Use KNATIVE_SERVING_RELEASE and replace the yaml name.

Reply:
I forgot it... Done.